.terraform-version

```diff
@@ -0,0 +1 @@
+0.10.1
```

Comment:
Yes, let's resolve it later (and add it back). Currently, there is a note about "0.9.6" in the code and this file says "0.10.1", so it is hard to guess which version is required, so worth documenting.